Integrations/Elasticsearch_v2/Elasticsearch_v2.py

```diff
@@ -118,6 +188,8 @@ def get_total_results(response_dict):
 
 
 def search_command():
+    """Performs a search in Elasticsearch.
```

Comment:
the closing """ should be in the same line for one liners, please fix that everywhere

Reply:
done - everywhere